1-exercises/A-accessing-values/exercise1.js

```diff
@@ -1,7 +1,7 @@
 /*
   This object has 4 properties
   The properties of the object are all primitive types (string, number or boolean)
-  What is the type of each property?
+  What is the type of each property? String
```

Comment:
Not all of the properties below are strings. The type of each property can be check by using the typeof operator.
Documentation for typeof https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof
Documentation for different js types https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures